sbol2/document.py

```diff
@@ -525,6 +516,9 @@ def parse_objects_inner(self, subject, obj):
         if subject not in self.SBOLObjects and obj in self.SBOL_DATA_MODEL_REGISTER:
             # Call constructor for the appropriate SBOLObject
             new_obj = self.SBOL_DATA_MODEL_REGISTER[obj]()
+            if isinstance(new_obj, Identified):
+                # Clear out the version. it will get set later
+                new_obj.version = ''
```

Comment:
I wondered if something like this might work...

Reply:
It is just one part of the recipe that allows versions to be properly handled.